Source/WebCore/platform/wpe/PlatformScreenWPE.cpp

```diff
@@ -92,6 +93,28 @@ bool screenSupportsExtendedColor(Widget*)
     return false;
 }
 
+bool screenSupportsHighDynamicRange(Widget* widget)
+{
+    std::string hdrCaps("false");
+
+    if(!widget)
+    {
+        return false;
+    }
+
+    // Get HDR capabilities of TV and STB
+    char *hdrCapsEnvValue = std::getenv("WPE_HDR_CAPABILITIES");
```

Comment:
In many existing files, we are comparing a value using strcmp and then return like below. With below, we dont need the local variable, assignment and comparison steps.
const char* disableGLSink = g_getenv("WEBKIT_GST_DISABLE_GL_SINK");
    if (disableGLSink && !strcmp(disableGLSink, "1")) {
        GST_INFO("Disabling hardware-accelerated rendering per user request.");
        return nullptr;
    }